integration/test_qca.in

```diff
@@ -0,0 +1,6 @@
+(FIND-QCA-PATH : TARGET ("SHC1") : SOURCE ("IRS1") : RELTYPE ("controls-state-change-of"))
+(SUCCESS :paths (((IRS1 (:93584034 (:directed True :interaction controls-state-change-of)) HRAS (:93581981 (:directed True :interaction controls-state-change-of)) RHOA (:93594410 (:directed True :interaction controls-state-change-of)) SHC1) (IRS1 (:93584046 (:directed True :interaction controls-state-change-of)) PIK3CA (:93590756 (:directed True :interaction controls-state-change-of)) RHOA (:93594410 (:directed True :interaction controls-state-change-of)) SHC1))))
```

Comment:
Is :93584034 an ID for the edge here? I think it would be better to use : for a keyword whose argument could be 93584034, for instance (:id 93584034 (:directed True ...))